app/src/main/java/com/globant/counter/android/mvp/view/CalculateView.java

```diff
@@ -0,0 +1,108 @@
+package com.globant.counter.android.mvp.view;
+
+import android.app.Activity;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.globant.counter.android.R;
+import com.squareup.otto.Bus;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * @author s.ruiz
+ */
+
+public class CalculateView extends ActivityView {
+
+    private final Bus bus;
+    private TextWatcher textWatcher;
+
+    @BindView(R.id.values)
+    EditText valuesToOperate;
+    @BindView(R.id.result)
+    TextView result;
+    private CharSequence lastInput;
+
+    public CalculateView(Activity activity, Bus bus) {
+        super(activity);
+        this.bus = bus;
+        ButterKnife.bind(this, activity);
+        addEditListeners();
```

Comment:
To detect text changes, you can use this ButterKnife annotation: @OnTextChanged.